src/pytest_mock_resources/cli.py

```diff
@@ -2,6 +2,12 @@
 import enum
 import subprocess  # nosec
 
+from pytest_mock_resources.config import get_env_config
+
+postgres_image = get_env_config("postgres", "image") or "postgres:9.6.10-alpine"
+mysql_image = get_env_config("mysql", "image") or "mysql:5.6"
+mongo_image = get_env_config("mongo", "image") or "mongo:3.6"
```

Comment:
we can go one step further:

if you instantiat the various configs, you can pull the image attr and it will have the same default (and leverage the env var in the cases where it is set.

```python
from pytest_mock_resources.container.postgres import PostgresConfig

postgres_image = PostgresConfig().image
```

Reply:
@oakhan3 done.